### PR TITLE
show link post domain after the link

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

#270 

#### What's this PR do?

this adds a little display next to the title link for a URL post which just shows what domain the link is from.

#### How should this be manually tested?

if you look at the frontpage or a channel page with link posts you should see something like:

fjfjasdf

the same thing should show up if you look at the comments for the post.

### CODE REVIEW

File: static/js/components/PostDisplay.js [hunk 1]
```diff
@@ -26,9 +26,14 @@ const postTitle = (post: Post) =>
     >
       {post.title}
     </Link>
-    : <a className="post-title" href={post.url} target="_blank">
-      {post.title}
-    </a>
+    : <div>
+      <a className="post-title" href={post.url} target="_blank">
```
Comment (noisecapella) [inline static/js/components/PostDisplay.js]: Can you add `rel="noopener noreferrer"` to this?

Comment (aliceriot) [reply static/js/components/PostDisplay.js]: weird that the linter didn't catch that...

**Comment by aliceriot** [comment]
@noisecapella addressed

**Comment by aliceriot** [comment]
ahaha I accidentally added it to the wrong branch https://github.com/mitodl/open-discussions/pull/296/commits/54fa169bff040b5424011ae5af32429a6a6d6cfe

:woman_facepalming:

File: static/js/components/PostDisplay.js (hunk 1 above)
Comment (noisecapella) [reply static/js/components/PostDisplay.js]: Would be a good use case for a custom eslint plugin

**Pull request merged**